### PR TITLE
Diff denom keeper tests

### PR DESCRIPTION
**Pull request by Josumner**
Added test fund with afet
Added test drip with stake and afet
- Just wondering what variable naming etiquette you use normally? I know the snake case + camel case combo doesn't really look very nice

### CODE REVIEW

File: x/airdrop/keeper/keeper_test.go [hunk 1]
```diff
@@ -164,6 +166,84 @@ func (s *KeeperTestSuite) TestFeeDrip() {
 	s.Require().Equal(feeCollectorBalance, sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(4040)))
 }
 
+func (s *KeeperTestSuite) TestAddNewFundWithDiffDenom() {
+	amount := sdk.NewInt64Coin("afet", 4000)
```
Comment (daeMOn63) [inline x/airdrop/keeper/keeper_test.go:170]: I'd vote for using another denom that `afet` here. This is test and can be anything really, no need to create any confusion with prod tokens.

File: x/airdrop/keeper/keeper_test.go [hunk 2]
```diff
@@ -164,6 +166,84 @@ func (s *KeeperTestSuite) TestFeeDrip() {
...
+
+	addrBalance = s.app.BankKeeper.GetBalance(s.ctx, addr1, "afet")
+	moduleBalance = s.app.BankKeeper.GetBalance(s.ctx, moduleAddress, "afet")
+
+	s.Require().Equal(addrBalance, sdk.NewCoin("afet", sdk.NewInt(0)))
+	s.Require().Equal(moduleBalance, sdk.NewCoin("afet", sdk.NewInt(4000)))
+}
+
+func (s *KeeperTestSuite) TestMultiDenomFeeDrip() {
+	amount_stake := sdk.NewInt64Coin(sdk.DefaultBondDenom, 4000)
```
Comment (daeMOn63) [inline x/airdrop/keeper/keeper_test.go:194]: on variable naming, go uses camelCase only.

**Review by daeMOn63 — CHANGES_REQUESTED** [review]

File: x/airdrop/keeper/keeper_test.go (hunk 1 above)
Comment (Josumner) [reply x/airdrop/keeper/keeper_test.go]: Yep, good call - let me know what you think of "denom" as a placeholder

**Review by daeMOn63 — DISMISSED** [review]

File: x/airdrop/keeper/keeper_test.go [hunk 3]
```diff
@@ -164,6 +166,84 @@ func (s *KeeperTestSuite) TestFeeDrip() {
...
+func (s *KeeperTestSuite) TestMultiDenomFeeDrip() {
+	amountStake := sdk.NewInt64Coin(sdk.DefaultBondDenom, 4000)
+	amountDenom := sdk.NewInt64Coin("denom", 4000)
+	fund1 := types.Fund{
+		Amount:     &amountStake,
+		DripAmount: sdk.NewInt(40)}
+
+	fund2 := types.Fund{
+		Amount:     &amountStake,
+		DripAmount: sdk.NewInt(4000), // will only last one block
```
Comment (ejfitzgerald) [inline x/airdrop/keeper/keeper_test.go:202]: It would be a slightly better test if we didn't use the same 4000 40 combination but mixed it up a little

**Review by ejfitzgerald — DISMISSED** [review]

**Comment by Josumner** [comment]
Sorry guys - my mistake! Accidentally pushed to this before merging and pushing to a new branch

**Review by daeMOn63 — APPROVED** [review]

**Pull request merged**